ArduPlane/Log.cpp

```diff
@@ -330,6 +366,15 @@ const struct LogStructure Plane::log_structure[] = {
       "PIQY", PID_FMT,  PID_LABELS, PID_UNITS, PID_MULTS },
     { LOG_PIQA_MSG, sizeof(log_PID),
       "PIQA", PID_FMT,  PID_LABELS, PID_UNITS, PID_MULTS },
+
+// @LoggerMessage: AETR
+// @Description: Normalised pre-mixer control surface outputs
+// @Field: TimeUS: Microseconds since system startup
+// @Field: Ail: Pre-mixer value for aileron output (between -4500 to 4500)
```

Comment:
As a future thing - I think we ought to avoid putting units and ranges into the field names.  If we really need them we can add them as an extra field - 'though we could possibly consider adding a non-SI unit for the -4500 to 4500 normalised range as we use it a lot.